tests/test_decorator.py

```diff
@@ -1,4 +1,6 @@
 import django
+from django.db.models import Value
+from django.db.models.functions import Concat
```

Comment:
![image](https://user-images.githubusercontent.com/318650/97602140-33e91900-19c8-11eb-810c-0e13aad1faac.png)

Looks like the order is not correct and `isort` would like to fix this as it is configured in a way that `import ...` statements have to be before `from ... import ...`.

There is a `pre-commit` hook in place, which you can use to automatically fix the file. It should run automatically on every commit after it had been installed using `poetry run pre-commit install`. However, you can also run it manually by running `poetry run pre-commit run -a` and it should automatically change the order of the import to the expected format.